libvips/foreign/pngload.c

```diff
@@ -81,9 +85,10 @@ static VipsForeignFlags
 vips_foreign_load_png_get_flags_source( VipsSource *source )
 {
 	VipsForeignFlags flags;
+	VipsForeignLoadPng *png = (VipsForeignLoadPng *) load;
```

Comment:
don't think this is right but i'm not sure how else to pass the unlimited flag down to `read_new` in `vips__png_isinterlaced_source`